docs/guides/adapting-new-services.md

```diff
@@ -46,15 +46,15 @@ then invokes a single service image. If setting up a more complex service chain
 
 
 # Requirements for Harmony Services
-In order for a service to run in Harmony several things need to be provided as covered in the following sections. A simple reference service, [harmony-service-example](https://github.com/nasa/harmony-service-example), provides examples of each. For examples of services that are being used in production see [harmony-netcdf-to-zarr](https://github.com/nasa/harmony-netcdf-to-zarr) and [podaac-l2ss-py](https://github.com/podaac/l2ss-py). This document describes how to fulfill these requirements in depth.
+In order for a service to run in Harmony, several things need to be provided as covered in the following sections. For examples of Harmony services and development guidance, see [Examples and Guidance](https://github.com/nasa/harmony-service-lib-py/blob/main/README.md#examples-and-guidance).
```

Comment:
This section refers right back to Harmony (circular reference).

Reply:
Yeah I noticed that too... but I think it makes sense for both documents to point to each other since there's some things you do in the harmony repo after you've setup your service using the service adapter. I'm not sure how I'd suggest changing it.

Reply:
I removed the harmony-service-lib-py reference back to this README to avoid the possible confusion.